Tests/MapboxNavigationTests/CarPlayManagerTests.swift

```diff
@@ -189,7 +189,8 @@ class CarPlayManagerTests: TestCase {
                 navigationStarted = true
             }
             
-            func carPlayManagerDidEndNavigation(_ carPlayManager: CarPlayManager) {
+            func carPlayManagerDidEndNavigation(_ carPlayManager: CarPlayManager,
+                                                byCanceling canceled: Bool) {
```

Comment:
Oh, good catch. We should also test that the old method gets called, in case we accidentally remove the redundant call in the future.

Reply:
Yep, improved tests to test both delegate methods.